debut de manager (tests git)

AlexisGado opened the pull request.

Commit:
tests

Commit:
nouvelle branche

AlexisGado committed:
Merge pull request #1 from MisterGado/squelette

merge squelette

adrien-le-franc commented:
Thank you for your contribution ! Before I accept your pull request, please translate everything into English and make sure that the code runs without errors.

AlexisGado committed:
First version of the manager

Il s'agit d'une première version (assez naïve) du manager qu'on a rapidement testé mais pas encore relié avec les joueurs.

adrien-le-franc commented:
outdated
